src/bugsnag.js

```diff
@@ -175,6 +225,181 @@
     }
   }
 
+  // Setup breadcrumbs for click events
+  function trackClicks() {
+    if(!getBreadcrumbSetting("autoBreadcrumbsClicks", true)) {
+      return;
+    }
+
+    window.addEventListener("click", function(event) {
```

Comment:
Would we need to do feature detection here to prevent this from throwing an error in older IE versions?